### ISSUE TITLE
'docutils' search at docs.python.org returns every section of the library reference

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#603]
From a message sent to webmaster@python.org:

```
From: "A.T.Hofkamp" <a.t.hofkamp@tue.nl>
To: webmaster@python.org
Subject: 'docutils' search at docs.python.org returns every section of the
 library reference
Date: Thu, 13 Jan 2011 10:05:20 +0100
X-Spambayes-Classification: ham; 0.02

Hello,

First of all, many thanks for creating such a nice website, I use it often, especially the online 
search at the docs.python.org site.

Today, I tried looking for 'docutils' there, and I got every section of the library reference.
I picked a section from it (9.6 "random" module), and my firefox browser did not find "docutils" in 
the text.

On closer inspection, that page does contain the word, namely in css tags like

<tt class="xref docutils literal">

It may be useful to teach the search engine not to use those tags.


Sincerely,
Albert
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/603
- Originally reported by: Anonymous
- Originally created at: 2011-01-13T15:29:42.665

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-15 08:56:23+00:00_

Indeed.  I'll have a look.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 19:32:24+00:00_

Closes #603: Do not use the HTML-ized title for building the search index (that
resulted in "literal" being found on every page with a literal in the
title).

→ <<cset 50209661dff1111f2d6e697a9cf7e6299535b476>>
